KEY.lua

```diff
@@ -18,8 +18,15 @@ function init()
 end
 
 function keyboard.code(code, value)
-  down = value == 0
-  buffer = code
+  down = value > 0
```

Comment:
key down (pressed) is when value is 1 (or in some rare instances 2)

Reply:
!